src/Servers/HttpSys/src/RequestProcessing/Request.cs

```diff
@@ -370,7 +370,7 @@ public async Task<X509Certificate2> GetClientCertificateAsync(CancellationToken
             var certLoader = new ClientCertLoader(RequestContext, cancellationToken);
             try
             {
-                await certLoader.LoadClientCertificateAsync().SupressContext();
+                await certLoader.LoadClientCertificateAsync();
```

Comment:
GetClientCertificateAsync is called from app code, you might still want the suppression here.

Reply:
We don't do it *anywhere* in ASP.NET Core, so this isn't special.